Makefile

```diff
@@ -14,6 +14,11 @@ ZWRAPDIR = zlibWrapper
 TESTDIR  = tests
 FUZZDIR  = $(TESTDIR)/fuzz
 
+# Define some install locations
```

Comment:
If these variables are only used by `cmake`,
maybe it's better for code maintenance to define them close to the place where they are used.